test/stancjs/stancjs.expected

```diff
@@ -25,27 +25,6 @@ model {
   y ~ normal(0, 1);
 }
 
-$ node data-generation.js
```

Comment:
Why did this go away?

Reply:
I couldn't figure out exactly why, but this test became non-deterministic in 4.12 and would therefore _always_ fail, even if you ran it multiple times in a row on the same machine and `promote`d each time. I believe the RNG that js_of_ocaml uses used to start with a fixed seed, but no longer does.

Reply:
Ok, I am cool with that answer, maybe lets just create an issue to investigate this? You can assign it to me, but wont get to it before next week.